content/spin/kv-store-tutorial.md

```diff
@@ -184,7 +168,7 @@ command = "tinygo build -target=wasi -gc=leaking -no-debug -o main.wasm main.go"
 
 ## Using the Spin SDK
```

Comment:
Maybe "Using the Key-Value Store" or "Write Code to Save and Load Data."  "Using the Spin SDK" doesn't really signal what we're going to do in this section.